int32[] servo_indices
float32[] torq
---
int32 err
